src/internal_events/vector.rs

```diff
@@ -2,18 +2,6 @@ use super::InternalEvent;
 use metrics::counter;
 use prost::DecodeError;
 
-#[derive(Debug)]
-pub struct VectorEventSent {
-    pub byte_size: usize,
-}
-
-impl InternalEvent for VectorEventSent {
-    fn emit_metrics(&self) {
-        counter!("processed_events_total", 1);
-        counter!("processed_bytes_total", self.byte_size as u64);
-    }
-}
-
```

Comment:
I assume this is unused and thus removed?

Reply:
Yep. More accurately `vector` sink was emitting this but the underlying `BytesSink` also emits those metrics so I removed the one in `vector` sink.

Reply:
`SocketEventsSent` is emitted in `BytesSink`.